vulnerabilities/models.py

```diff
@@ -218,6 +218,15 @@ def with_vulnerability_counts(self):
         )
 
 
+def get_purl_query_lookups(purl):
```

Comment:
Dostring would be nice, with some explanations about why qualifiers and subpath are not included.